client/stylesheets/_toCat.scss

```diff
@@ -831,7 +832,7 @@ input[type="search"]::-webkit-search-results-decoration { display: none; }
 
 .searchfield {
     text-align: center;
-    margin: -20px 0 10px 0;
+    margin: -20px auto 10px auto !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Shorthand form for property `margin` should be written more concisely as `-20px auto 10px` instead of `-20px auto 10px auto`